spec/services/onestop_id_spec.rb

```diff
@@ -1,64 +1,50 @@
+# Create a concrete class for testing
+class TestOnestopId < OnestopId::OnestopIdBase
+  PREFIX = :s
+  MODEL = Stop
+  NUM_COMPONENTS = 3
+end
+
 describe OnestopId do
   it 'fails gracefully when given an invalid geometry' do
     expect {
-      OnestopId.new('Retiro Station', '-58.374722 -34.591389')
+      TestOnestopId.new('Retiro Station', '-58.374722 -34.591389')
     }.to raise_error(ArgumentError)
 
     expect {
-      OnestopId.new('Retiro Station', [-58.374722,-34.591389])
+      TestOnestopId.new('Retiro Station', [-58.374722,-34.591389])
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.